app.js

```diff
@@ -1,28 +1,24 @@
+//packages
 var Discord = require("discord.js")
-var bot = new Discord.Client();
+var client = new Discord.Client();
+var ffmpeg = require("ffmpeg-binaries")
+//configuration stuff
+const {apikey} = require("./config");
+const {prefix} = require("./config");
+client.login(apikey);
 
-const {apikey} = require("./api-key");
 
+//commands
+const {join} = require("./commands/basic/join");
+const {music} = require("./commands/music/music");
```

Comment:
same

Reply:
same what? Im confused.

Reply:
same as previous comment, whatever that was.